osquery/tables/system/windows/logged_in_users.cpp

```diff
@@ -119,6 +119,31 @@ QueryData genLoggedInUsers(QueryContext& context) {
       r["host"] = std::string(addr, CLIENTADDRESS_LENGTH);
     }
 
+
+    if (!r["host"].length()) {
+      LPSTR clientName = nullptr;
+      res = WTSQuerySessionInformationA(WTS_CURRENT_SERVER_HANDLE,
+                                        pSessionInfo[i].SessionId,
+                                        WTSClientName,
+                                        &clientName,
+                                        &bytesRet);
+
+      if (res == 0 || clientName == nullptr) {
+        VLOG(1) << "Error querying WTS clientName information (" << GetLastError()
+                << ")";
+        results.push_back(r);
+        WTSFreeMemory(clientName);
```

Comment:
Does it make sense to free this, since it's in a `clientName == nullptr` block?

Reply:
Right, this is non sense, will fix tomorrow